lbry/file/file_manager.py

```diff
@@ -81,8 +81,11 @@ async def download_from_uri(self, uri, exchange_rate_manager: 'ExchangeRateManag
         payment = None
         try:
             # resolve the claim
-            if not URL.parse(uri).has_stream:
-                raise ResolveError("cannot download a channel claim, specify a /path")
+            try:
+                if not URL.parse(uri).has_stream:
+                    raise ResolveError("cannot download a channel claim, specify a /path")
+            except ValueError:
+                raise InvalidURLError(uri)
```

Comment:
The other raised `ResolveError` should also probably be `InvalidURLError`.

Reply:
Well, that was already there, so I didn't touch it.

I can change it to `InvalidURLError`, but if it was me, I would create another exception like `NotDownloadableError` as a channel is a valid URL but cannot be downloaded, so using a more precise exception would be better.